### PR TITLE
Topic add on copy

### PR DESCRIPTION
**Pull request by devinblais**
Add an `onCopy` prop to the editor that fires on copy!
Also added support to run on Node10

### CODE REVIEW

File: src/component/handlers/edit/editOnCopy.js [hunk 1]
```diff
@@ -39,6 +39,10 @@ function editOnCopy(editor: DraftEditor, e: SyntheticClipboardEvent): void {
     const clipboardDataToSet = editor.props.convertBlockMapToClipboard(fragment);
     setClipboardData(e, editor, clipboardDataToSet);
   }
+
+  if (editor.props.onCopy) {
+      editor.props.onCopy(fragment);
```
Comment (max-winderbaum) [inline src/component/handlers/edit/editOnCopy.js]: Seems like to do what you want you'd want the full editorState and the selection, no?

Comment (devinblais) [reply src/component/handlers/edit/editOnCopy.js]: We grab all that already in @textio/editor (for our other onCopy function). Going for consistency. Thoughts?

Comment (max-winderbaum) [reply src/component/handlers/edit/editOnCopy.js]: I'm just not sure if the one in @textio/editor will be synced with this state given we're in async hell at that point... I guess now that I'm looking at it it *does* grab it from `editor._latestEditorState` which tries to be consistent...

up to you /shrug

**Review by max-winderbaum — APPROVED** [review]
This might work or it might break everything - let's find out 🎲 

(only kidding - looks good)

**Review by fraziermork — APPROVED** [review]
LGTM!

**Pull request merged**